resources/object_repo/login_obj.resource

```diff
@@ -19,6 +19,7 @@ ${lblPageTitle}             //span[contains(@data-test-id,"resource-title") and
 ${elemDataServiceNav}       //li[starts-with(.,"Data Services") and contains(@class,"nav")]
 ${elemDataAccessNav}        //li[starts-with(.,"Data Services") and contains(@class,"nav")]/section//li[contains(.,"Database Access")]
 ${btnPerspective}           (//div[@id="page-sidebar"]//button)[1]
+${elemCurrPerspective}      //button[contains(@class,"dropdown")]//h2[.="<<perspective>>"]
```

Comment:
Can you explain what this object is?

Reply:
The xpath is for the Current Perspective of the page, could either be Administrator or Developer
Basically it references this text

![image](https://user-images.githubusercontent.com/100675023/197185328-1c48804f-9f8b-4dfb-951f-051782b16802.png)